templates/agenda_case_item/inspection.html.twig

```diff
@@ -0,0 +1,101 @@
+{% extends 'layout-with-agenda-case-item-submenu.html.twig' %}
+
```

Comment:
Maybe a todo comment in this would be helpful, so we know we have to revisit it.

Reply:
Added `TODO`.